src/interface.jl

```diff
@@ -73,7 +80,14 @@ transform!(b, x, y) = copyto!(y, transform(b, x))
 
 Return `log(abs(det(J(b, x))))`, where `J(b, x)` is the jacobian of `b` at `x`.
 """
-logabsdetjac(b, x) = last(with_logabsdet_jacobian(b, x))
+function logabsdetjac(b, x)
+    res = with_logabsdet_jacobian(b, x)
+    if res isa ChangesOfVariables.NoLogAbsDetJacobian
+        error("`logabsdetjac` not implemented for $(typeof(b)); implement `logabsdetjac` and/or `with_logabsdet_jacobian`.")
```

Comment:
Same here, maybe rather say that with_logabsdet... is used in the fallback for logabsdetjac but not defined?